addons/web/__manifest__.py

```diff
@@ -99,6 +99,7 @@
             'web/static/src/search/**/*',
             'web/static/src/views/**/*',
             'web/static/src/webclient/**/*',
+            ('remove', 'web/static/src/webclient/navbar/navbar.scss'), # already in _assets_common_styles
```

Comment:
I don't understand this ? It's in_ssets_common_styles indeed but that is not in the backend assets so it does not have to be removed ? :thinking: Note: I don't understand why that _assets_common_styles exists at all to be honest, it is only included once... by the common assets :shrug: 

Anyways I guess this will go away anyway since ADR is getting rid of the common assets bundle I guess.